### PR TITLE
update examples/cassandra to api v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
related: #6584

### CODE REVIEW

**Comment by nikhiljindal** [comment]
LGTM, thanks!

File: examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public KubernetesSeedProvider(Map<String, String> params) {
 
         String host = protocol + "://" + hostName + ":" + hostPort;
         String serviceName = getEnvOrDefault("CASSANDRA_SERVICE", "cassandra");
-        String path = "/api/v1beta1/endpoints/";
+        String path = "/api/v1beta3/endpoints/";
```
Comment (thockin) [inline examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java]: This is buggy.  I ran across it looking at something else.  You can't just change the API version and not the calling code.  Look 7 lines down - it is trying to decode this object into a v1beta1 structure.

Please audit this and other v1beta3 conversions for this same problem.

Comment (bgrant0607) [inline examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java]: @caesarxuchao Did you run the example to ensure that it worked?

Comment (bgrant0607) [inline examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java]: The only other URL I found was in rethinkdb, which hasn't been converted yet.

Good catch, thanks.

**Comment by caesarxuchao** [comment]
I ran through the tutorial, but I didn't check if Cassandra worked properly. I submitted a PR for rethinkdb before I left today. I will ask @nikhiljindal to hold that PR.

File: examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java (hunk 1 above)
Comment (caesarxuchao) [inline examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java]: I ran through the tutorial, but I didn't check if Cassandra worked properly. I submitted a PR for rethinkdb before I left today. I will ask @nikhiljindal to hold that PR.

**Pull request merged**